### DISCUSSION TITLE
rmarkdown code chunk options for tar_visnetwork

### DISCUSSION THREAD

**Discussion by softloud** [ropensci/targets#819] [Help]
I'm using `targets::` to illustrate [how to patch and extend](https://softloud.github.io/spunk/extending.html) a living open-source Cochrane analysis. Subset pipeline visualisations are wonderful for this, but I keep running to this problem in my `distill::` articles. How do I suppress this output before the network visualisation? Is there a codechunk option I can use, or an argument in the function?

![image](https://user-images.githubusercontent.com/13408706/162628356-df5f267f-a0b5-40b6-a690-be63867d2a0e.png)

**Comment by wlandau** [comment]
You could set the `callr_arguments` argument to `list(spinner = FALSE)`. If that does not work, then `list(spinner = FALSE, show = FALSE)` should.

**Reply by softloud** [reply]
Tak, as they say here in Denmark :smile: That did it. Much appreciated.
